variables.tf

```diff
@@ -131,3 +131,48 @@ variable "tags" {
   type    = map(string)
   default = {}
 }
+
+
+variable "vnet_name" {
+  description = "Vnet Name for Private Subnets"
+  type        = string
+  default     = ""
+}
+
+variable "vnet_rg_name" {
+  description = "Vnet Resource Group Name for Private Subnets"
+  type        = string
+  default     = ""
+}
+
+variable "subnet_cidr" {
+  description = "Vnet Subnet CIDR"
+  type    = list(string)
+  default     = []
+}
+
+variable "create_subnet" {
+  description = "Should Create Subnet"
+  type        = bool
+  default     = false
+}
+
+
+variable "revision" {
+   description = "Revision of Functional App"
+   type        = string
+   default     = ""
+}
+
+variable "namespace" {
```

Comment:
don't think this variable is needed for function app

Reply:
Ok removing namespace..